src/components/calculator.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import './calculator.css';
+
+export default class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="container">
+        <span className="result">0</span>
+        <span className="btn">AC</span>
+        <span className="btn">+/-</span>
+        <span className="btn">%</span>
+        <span className="btn sign-color">/</span>
```

Comment:
- [ ] Kindly use the correct division symbol here. Look at the original design for comparison. :pray: 

![image](https://user-images.githubusercontent.com/64894927/134204424-75bf2535-cbc8-4cca-aece-20dc8f4c73e7.png)